integration-test-remote/src/test/java/io/cdap/cdap/app/etl/google/drive/GoogleDriveTest.java

```diff
@@ -89,6 +108,7 @@
   private static final String TEST_TEXT_FILE_CONTENT = "text file content";
   private static final String TEST_DOC_FILE_CONTENT = "Google Document file content";
   private static final String TEST_SHEET_FILE_CONTENT = "a,b,c\r\n,d,e";
+  public static final String TMP_FOLDER = "/tmp/googleDriveTestFolder";
```

Comment:
cdap can run on windows. Please use
`Files.createTempDirectory` instead. Also don't forget to close it.

Reply:
done